scripts/ckan_migrate/organizations.py

```diff
@@ -0,0 +1,81 @@
+import logging
+from .helpers import ensure_unique_name
+
+log = logging.getLogger(__name__)
+
+FIELDS = [
+    'id', 'name', 'title', 'description', 'created', 'state', 'type',
+    'approval_status', 'image_url', 'is_organization'
+]
+
+
+def _transform(row):
+    return {
+        'id': row['id'],
+        'name': row['name'],
+        'title': row.get('title'),
+        'description': row.get('description'),
+        'created': row.get('created'),
+        'state': (row.get('state') or 'active'),
+        'type': (row.get('type') or 'organization'),
+        'approval_status': row.get('approval_status'),
+        'image_url': row.get('image_url'),
+        'is_organization': True,
+    }
+
+
+def _upsert(new_db, gdict):
+    placeholders = ', '.join(['%s'] * len(FIELDS))
+    set_cols = ', '.join(FIELDS)
+
+    new_db.cursor.execute('SELECT 1 FROM public."group" WHERE id = %s', (gdict['id'],))
+    if new_db.cursor.fetchone():
+        sql = f'UPDATE public."group" SET ({set_cols}) = ({placeholders}) WHERE id = %s'
+        new_db.cursor.execute(sql, tuple(gdict[f] for f in FIELDS) + (gdict['id'],))
+    else:
+        sql = f'INSERT INTO public."group" ({set_cols}) VALUES ({placeholders})'
+        new_db.cursor.execute(sql, tuple(gdict[f] for f in FIELDS))
+
+
+def import_organizations(old_db, new_db):
```

Comment:
Use import_groups for all
Is there any specific reason why we need it to be separate?